packages/components/app/styles/components/code-block/theme.scss

```
@@ -0,0 +1,155 @@
+// THEMING
+
+.hds-code-block--theme-dark {
+  // COLORS
+
+  // Descriptive color names:
+
+  --code-block-color-blue: #2d8eff;
+  --code-block-color-cyan: #32fff7;
+  --code-block-color-matrix-green: #86ff13;
+  --code-block-color-orange: #ffa800;
+  --code-block-color-purple: #c76cff;
+  --code-block-color-red: #ff3b20;
+
+  // Semantic color names:
+
+  // Base UI colors:
+  --code-block-color-foreground-primary: #d5d7db;
+  --code-block-color-foreground-faint: #b2b6bd;
+  --code-block-color-surface-primary: #0d0e12;
+  --code-block-color-surface-faint: #15181e;
+  // status ----
+  --code-block-color-foreground-success: #009241;
+  --code-block-color-foreground-critical: #ef3016;
+  // interactive ---
+  --code-block-color-foreground-interactive: #dedfe3;
+  --code-block-color-foreground-interactive-hover: #fff;
+  --code-block-color-foreground-interactive-active: #f1f2f3;
+  // actions ---
+  --code-block-color-foreground-action-default: #2b89ff;
+  --code-block-color-foreground-action-hover: #389aff;
+  --code-block-color-foreground-action-active: #4ca1ff;
+  // borders (TODO: Names are confusing) ----
+  --code-block-color-border-strong: rgba(178, 182, 189, 40%); // #b2b6bd66
+  --code-block-color-border-primary: rgba(178, 182, 189, 20%); // #b2b6bd33
+
+  // CodeBlock UI:
+  // lines of code - highlighted ----
+  --code-block-color-line-highlight: rgba(0, 74, 222, 30%);
```

I may have tweaked this value a bit without communicating to you 😢 
```suggestion
  --code-block-color-line-highlight: rgba(0, 74, 222, 20%);
```